service/src/main/kotlin/fi/espoo/evaka/attachment/AttachmentsController.kt

```diff
@@ -147,9 +147,11 @@ class AttachmentsController(
         val attachment =
             db.read { it.getAttachment(attachmentId) ?: throw NotFound("Attachment $attachmentId not found") }
 
+        val extension = parseExtension(attachment.name) ?: throw Forbidden("Couldn't parse valid file extension for attachment $attachmentId")
```

Comment:
I don't think it's necessary to throw on "invalid" extension since the validity should have been checked when the attachment was saved. At this point we should just assume that whatever the extension is, it should be valid.